website/docs/r/app_configuration.html.markdown

```diff
@@ -220,6 +235,14 @@ An `identity` block exports the following:
 
 ---
 
+A `replica` block exports the following:
+
+* `id` - The ID of the App Configuration Replica.
+
+* `endpoint` - The URL of the App Configuration Replica.
+
```

Comment:
This change hasn't been transferred to the data source documentation?

Reply:
fixed, thanks.